examples/diffusion/python_controlnet_canny_sd_15/README.md

```diff
@@ -17,7 +17,7 @@ python3 -m venv sd_venv
 Install dependencies
 
 ```bash
-pip install -r requirements.txt
+pip install -r torch_requirements.txt -r requirements.txt
```

Comment:
See https://pip.pypa.io/en/stable/reference/requirements-file-format/
You can point to `torch_requirements.txt` from `requirements.txt`. If you can get that to work so that this command is simply `pip install -r requirements.txt`, that would be better imo

Reply:
Looking into it more, this wont actually allow it to use different index-url parameters just by chaining requirement files. Theres probably some way to do this with --extra-index-url flags and adding versions but that will just make it more complicated.